### ISSUE TITLE
Scroll jumps to the last vueper-slides on page open misbehaviour

### ISSUE THREAD

**Issue by rucebee** [antoniandre/vueper-slides#96]
When page loads it jump to last vueper-slides - for landing page it's a disaster: focus should stay on top

https://toyabox.com/

Vue:
```
<div class="img">
  <vueper-slides fixed-height="true">
    <vueper-slide image="/images/quest-fairy-1.jpg"/>
    <vueper-slide image="/images/quest-fairy-2.jpg"/>
    <vueper-slide image="/images/quest-fairy-3.jpg"/>
  </vueper-slides>
</div>
```

CSS:
```
  .img {
    position: relative;

    .vueperslides {
      position: absolute;
      left: 0;
      top: 0;
      height: 100%;
      width: 100%;
    }
  }
```

**Comment by antoniandre** [comment]
Hi @rucebee, this is not intended without user action.
Should now be fixed in version 2.9.1.

**Comment by rucebee** [comment]
perfect, super speed fixing))
